packages/commonwealth/client/scripts/views/pages/CommunityManagement/Integrations/Snapshots/Snapshots.tsx

```diff
@@ -60,14 +60,16 @@ const Snapshots = () => {
         <CWText type="h4">Snapshot</CWText>
         <CWText type="b1">
           You can connect multiple Snapshot spaces to keep your voting
-          conversations all in one place.
+          conversations all in one place. Note: only input the name of your
+          snapshot space, not the full url.
```

Comment:
Are we able to handle either for them? Would prefer if this were handled intelligently, rather than putting it on the user.

Reply:
@CowMuon Yes I can set up input validation so that they aren't able to input an address with http or https. Good catch.